relays/ethereum/src/substrate_client.rs

```diff
@@ -131,6 +128,7 @@ pub async fn ethereum_header_known(
 /// Submits Ethereum header to Substrate runtime.
 pub async fn submit_ethereum_headers(
 	client: Client,
+	signer: sp_core::sr25519::Pair,
```

Comment:
That should probably be configurable, but not in this PR. Could you log an issue?

Reply:
It is already configurable (`--sub-signer=//Alice --sub-signer-password=HelloWorld`)

Reply:
I meant the crypto type (either `sr` or `ed`). I don't think it's required for all chains to use `MultiSigner` and support both crypto.

Reply:
Oh, ok - I see now, thanks. But relay directly references (our) runtime - I mean to encode `Call`s, generate signatures, etc. And our runtime specifies that it is using `MultiSignature`. So I'm not sure it'll be easy to make it generic over runtime. Wdyt?

I had similar problem in secret-store and I ended up with moving everything runtime-specific to separate file ([see here](https://github.com/paritytech/secret-store/blob/master/bin/substrate/src/runtime.rs#L17)). So to reconfigure SS for different chain, you just need to redefine what's require in this file. We could do something like that here, or do you think there's a better way?

Reply:
Yeah, I think that's fine, it's super low-priority though. I'm almost certain that nearly all of the chains will be using `MultiSigner` so we should be fine with single, hardcoded crypto.

Reply:
#61